string pcloud_filename
string image_filename
---
bool success
